core/src/schema/inference/mod.rs

```diff
@@ -267,6 +269,7 @@ impl MergeStrategy<number_content::I64, i64> for OptionalMergeStrategy {
             number_content::I64::Range(range) => self.try_merge(range, candidate),
             number_content::I64::Categorical(cat) => self.try_merge(cat, candidate),
             number_content::I64::Constant(cst) => self.try_merge(cst, candidate),
+            I64::Id(id) => self.try_merge(id, candidate)
```

Comment:
```suggestion
            number_content::I64::Id(id) => self.try_merge(id, candidate)
```
as a result of the weird import above.